### ISSUE TITLE
Don't download all race data every two minutes. Only grab what is new.

### ISSUE THREAD

**Issue by ThomasThoren** [TheLens/elections#222]
If we do this, we could also increase the frequency that the front end checks for new data to every minute. The Secretary of State uploads new results about every two minutes during the busiest part of the night.

This could be accomplished by uploading a single JSON file with the most recent Secretary of State data's timestamp, similar to that used in force_reload.json. Every minute, the front end can check this file. If the timestamp has changed, then download new data. If the timestamp is the same, don't do anything.

Checking every minute will provide the most up-to-date data possible without requiring much additional data transferred since that new JSON file will be tiny (25 bytes or so).

**Comment by myersnews** [comment]
yeah, that’s best. under the current setup we could be as much as 3:59 behind SoS, if we check for a new file and immediately after the SoS posts a new file.


--
Steve Myers
smyers@thelensnola.org
Special projects editor, The Lens
504-298-9750
@myersnews

**Comment by ThomasThoren** [comment]
Added back-end and front-end code for this. Waiting for the Secretary of State's test to see if it holds up.
